controllers/cluster_controller_legacy.go

```diff
@@ -60,6 +60,13 @@ func NewClusterReconcilerLegacy(client client.Client, log logr.Logger, scheme *r
 // +kubebuilder:rbac:groups="",namespace=eksa-system,resources=secrets,verbs=delete;
 // +kubebuilder:rbac:groups=tinkerbell.org,resources=hardware;hardware/status,verbs=get;list;watch;update;patch
 // +kubebuilder:rbac:groups=bmc.tinkerbell.org,resources=machines;machines/status,verbs=get;list;watch
+//
+// For the full cluster lifecycle to support Curated Packages, the controller
+// must be able to create, delete, update, and patch package bundle controller
+// resources, which will trigger the curated packages controller to do the
+// rest.
+//
+// +kubebuilder:rbac:groups=packages.eks.amazonaws.com,resources=packagebundlecontrollers,verbs=create;delete;get;list;patch;update;watch;
```

Comment:
What is the purpose of adding this comment then? Generally we don't need to backport to legacy controller anyways.

Reply:
This was the only place I found to add these comments that was actually reflected in the final output. Can you suggest a better place to put them?

Reply:
This file is for the old controller, so I would suggest to try in cluster_controller.go. Or if it's not applicable then omit.

Reply:
I'll try again, but the last time I made modifications there, they plain just didn't show up... Which seemed weird, but maybe I was using the wrong makefile target... or something.